bin/edbgWrapper.sh

```diff
@@ -5,10 +5,10 @@
 # ***************************************************************************
 cmd=${0##*/}
 
-# ***************************************************************************
-# Figure out what the user's ecmd executable is set to, if they have one.
-# ***************************************************************************
-if [ ! -x "$ECMD_EXE" ]
+ECMD_EXE="/usr/bin/edbg"
```

Comment:
You cannot hard code paths in this file.  This breaks all other users of this file.  That applies to the ECMD_DLL_FILE below as well.

Reply:
The path is exported only if the file is present in that path else it is going to take default logic of checking environment variable

Reply:
That is SUPER unclear in the code.  You are using a quirk of shell where a local variable and an exported variable can have different values.  What is going on is not at all clear in an initial reading of the code if you aren't aware of that.  That's why I missed it the first time until I thought thru it more.

So, at a minimum, the variable naming needs to be fixed and comments explain the change added.  But also, this is not the right solution for this issue as has been discussed in other forums.

Additionally, a PR should be focused on fixing the issue in the description.  This change isn't related to that in anyway.

Reply:
I will update the comments and address what you requested. 

I am fixing mainly two issues in this PR

1. An offline build issue in OpenBMC yocto (That's the meson.build is concentrated on) 
2. Environment variable settings (That's this wrapper script change is concentrated on)

Reply:
You can fix the first one in this PR, but inclusion of the 2nd one will cause this entire PR to be rejected.  There is still the other discussion going on for how to best handle environment variables.  Until that is resolved, I am not merging half fixes.

Reply:
I will push the environment-related one as seperate PR